RecoJets/JetProducers/python/PileupJetIDParams_cfi.py

```diff
@@ -1,6 +1,212 @@
 import FWCore.ParameterSet.Config as cms
 from RecoJets.JetProducers.PileupJetIDCutParams_cfi import *
 
+####################################################################################################################
+full_102x_chs = cms.PSet(
+        impactParTkThreshold = cms.double(1.),
+        cutBased = cms.bool(False),
+        etaBinnedWeights = cms.bool(True),
+        tmvaMethod = cms.string("JetIDMVAHighPt"),
+        version = cms.int32(-1),
+        nEtaBins = cms.int32(4),
+        trainings = cms.VPSet(
+            cms.PSet(
+                jEtaMin = cms.double(0.0),
+                jEtaMax = cms.double(2.5),
+                tmvaWeights  = cms.FileInPath("RecoJets/JetProducers/data/pileupJetId_102X_Eta0p0To2p5_chs_BDT.weights.xml.gz"),
+                tmvaVariables = cms.vstring(
+                    "nvtx"      ,
+                    "dR2Mean"   ,
+                    "nParticles",
+                    "nCharged"  ,
+                    "majW"      ,
+                    "minW"      ,
+                    "frac01"    ,
+                    "frac02"    ,
+                    "frac03"    ,
+                    "frac04"    ,
+                    "ptD"       ,
+                    "beta"      ,
+                    "pull"      ,
+                    "jetR"      ,
+                    "jetRchg"   ,
+                )
+                ),
+            cms.PSet(
+                jEtaMin = cms.double(2.5),
+                jEtaMax = cms.double(2.75),
+                tmvaWeights   = cms.FileInPath("RecoJets/JetProducers/data/pileupJetId_102X_Eta2p5To2p75_chs_BDT.weights.xml.gz"),
+                tmvaVariables = cms.vstring(
+                    "nvtx"      ,
+                    "dR2Mean"   ,
```

Comment:
it looks like the same variable name vector repeats for the first 3 ranges. These can be initialized from a separately defined (temporary) vector

Reply:
True, I will change it